### PR TITLE
perf: me template update

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: templates/scenarios/csharp/message-extension/infra/azure.parameters.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "resourceBaseName": {
+        "value": "nonssome${{RESOURCE_SUFFIX}}"
```
Comment (xzf0587) [inline templates/scenarios/csharp/message-extension/infra/azure.parameters.json:6]: mebot is better?

File: templates/scenarios/csharp/message-extension/teamsfx/app.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+version: 1.0.0
+
+provision:
+  - uses: botAadApp/create # Creates a new AAD app for Azure Bot service if BOT_ID environment variable is empty
+    with:
+      name: notification-bot
```
Comment (xzf0587) [inline templates/scenarios/csharp/message-extension/teamsfx/app.yml:6]: update the name

File: templates/scenarios/csharp/message-extension/infra/azure.parameters.json (hunk 1 above)
Comment (wenytang-ms) [reply templates/scenarios/csharp/message-extension/infra/azure.parameters.json]: thanks, updated as mebot as prefix

File: templates/scenarios/csharp/message-extension/teamsfx/app.yml (hunk 2 above)
Comment (wenytang-ms) [reply templates/scenarios/csharp/message-extension/teamsfx/app.yml]: thanks, rename as me-bot

**Review by xzf0587 — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**